clang/lib/CodeGen/CGOpenMPRuntime.cpp

```diff
@@ -60,6 +60,8 @@ class CGOpenMPRegionInfo : public CodeGenFunction::CGCapturedStmtInfo {
     ParallelOutlinedRegion,
     /// Region with outlined function for standalone 'task' directive.
     TaskOutlinedRegion,
+    /// Region with outlined function for standalone 'taskgraph' directive.
+    TaskgraphOutlinedRegion,
```

Comment:
We need to generate a function out of the taskgraph's associated statement. It is required since the function pointer and arguments are later passed to the runtime to decide if the captured region should be executed or not (depending if the taskgraph is already recorded).